osu.Game/Screens/SelectV2/SongSelect.cs

```diff
@@ -348,6 +372,14 @@ public void RequestDeleteBeatmap(BeatmapSetInfo set)
             dialogs?.Push(new BeatmapDeleteDialog(set));
         }
 
+        /// <summary>
+        /// Opens up <see cref="BeatmapClearScoresDialog"/> with the given beatmap.
+        /// </summary>
+        public void RequestClearScores(BeatmapInfo beatmap)
+        {
+            dialogs?.Push(new BeatmapClearScoresDialog(beatmap));
```

Comment:
Contrary to old song select, I'm intentionally not refreshing the leaderboard on completion of clearing scores. Old song select code:

https://github.com/ppy/osu/blob/b31e68538037f7272b7707d9b55584dfc91268dd/osu.Game/Screens/Select/SongSelect.cs#L1055-L1057

I have manually tested and confirmed that this is not needed.